create_secrets.py

```diff
@@ -13,10 +13,13 @@
 
 
 def set_databse_urls(secrets):
-    write_secrets['ckan'].append(
-        'export SQLALCHEMY_URL=postgresql://ckan:{db_password}@db/ckan'.format(
+    alchemy_url = 'SQLALCHEMY_URL=postgresql://ckan:{db_password}@db/ckan'.format(
             db_password=secrets['db-POSTGRES_PASSWORD']
-    ))
+            )
+    write_secrets['ckan'].append('export {}'.format(alchemy_url))
+    # add as env file also
+    write_secrets['ckanenv'].append(alchemy_url)
```

Comment:
What is this new file?
Why not name it with a non-confusing name (i.e. 'harvester')?

Reply:
This file include the CKAN secrets as _env_ file (without the _EXPORT_). Use the name you prefer. This file could be used for any container, no just harvester.

Reply:
If it's only used by harvester, then name it accordingly.
e.g. the ckan secrets are named `ckan` as they're being used for ckan

Reply:
Changed for _harvester_